test/Tests/Writers/HTML.hs

```diff
@@ -30,6 +33,11 @@ infix 4 =:
      => String -> (a, String) -> TestTree
 (=:) = test html
 
+infix 4 =::
+(=::) :: (ToString a, ToPandoc a)
+     => String -> (a, String) -> TestTree
+(=::) = test htmlQTags
+
```

Comment:
I think that for code clarity it would be better not to define a special-purpose operator for this; just use `test htmlQTags` to avoid confusion down the line...

Reply:
Yeah I did that originally but I got an error saying the type was to generic. Hence I needed to add a bound in some way to limit the possible types. I figured I would do the same as done before. I could try to see if I get it working another way, but I need some way to limit the types hence I think I will need a function declaration no matter what.

Reply:
Probably you just need to add a type declaration somewhere to the argument of `test htmlQTags` (using `ScopedTypeVariables` extension).
Or use a function declaration, as you are, but I think it's cleaner if we reserve the operators for really generic stuff.

Reply:
Ended up using a simple function declaration because I didn't figure out how to solve it using `ScopedTypeVariables`.